ruby_oop/mastermind/display_text.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# module that has the words displayed to the screen
+module DisplayableText
```

Comment:
These texts are static so they can be simpler. Also `puts` can be removed from the module. 

They can be a method to return a string.

```
  def codemaker_confirmation_text
    'You are the codemaker.'
  end
```

Then the user of the module may do something like

```rb
include DisplayableText

puts codemaker_confirmation_text
```

Alternatively these texts can be defined as constants.

```rb
CODEMAKER_CONFIRMATION_TEXT = 'You are the codemaker.'.freeze
```

Then the user of the module may do something like

```rb
puts DisplayableText::CODEMAKER_CONFIRMATION_TEXT
```